scripts/lib/libvirt/admin-config

```diff
@@ -18,6 +18,10 @@ def main():
                         help="Local Repository Directory Source")
     parser.add_argument("localrepotgt",
                         help="Local Repository Directory Target")
+    parser.add_argument("firmwaretype",
```

Comment:
If there are close set of choices allowed, I would add the `choices` parameter: https://docs.python.org/3/library/argparse.html#choices